### DISCUSSION TITLE
Add feature : option to remove screen confirm full payment

### DISCUSSION THREAD

**Discussion by hogsim** [Blair2004/NexoPOS#540] [Ideas]
Hi, 

Can you add an option in the settings to remove this screen and consider it's a yes automatically ? 

![image](https://user-images.githubusercontent.com/5645306/150549294-b962de69-e97c-4759-bb64-c34edcddf43b.png)

**Comment by Blair2004** [comment]
Hi

We might consider this if it's requested by many users. Vote needed here.

Thanks for your suggestions.

**Comment by Blair2004** [comment]
After further internal discussion, we believe the full payment requires confirmation. We'll move this to the ideas where other votes can occur.

**Reply by boudo2017** [reply]
Only full due amount or partial amount needs confirmation, 
These required also a registered customer only.
Walk-in customers have to pay the full amount.
-full payment do not need that.
That is how it works in the whole shops in the world.
Thank you
